lib/media/track/remotedatatrack.js

```diff
@@ -22,42 +21,30 @@ const Track = require('./');
  *   null. In other words, if this is true, there is no bound on packet lifetime
  *   or the number of retransmits that will be attempted, ensuring "reliable"
  *   transmission.
- * @property {Track.SID} sid - The {@link RemoteDataTrack}'s SID
  * @emits RemoteDataTrack#message
  * @emits RemoteDataTrack#unsubscribed
  */
 class RemoteDataTrack extends Track {
```

Comment:
Should this extend some RemoteTrack? Otherwise, I don't see where `sid` and `isSubscribed` are going to live.

Reply:
Nevermind—I see how it's handled below.